tox.ini

```diff
@@ -496,7 +462,7 @@ recreate = true
 deps =
 
 commands =
-    pip freeze
```

Comment:
the `dependencies` seems to me like overengineering .... create a new virtualenv just to run pip freeze ? 

Is the list of dependencies reviewed on a regular basis?

Reply:
Yes, I use this to update the dependency list periodically.